src/WorkBC.Web/ClientApp/projects/jb-lib/src/lib/filters/models/job.model.ts

```diff
@@ -170,7 +170,7 @@ export class Job extends RecommendedJob {
     if (savedJob) {
       this.City = savedJob.city;
       this.DatePosted = savedJob.datePosted.toString();
-      this.ExpireDate = savedJob.expireDate.toString();
+      this.ExpireDate = savedJob.expireDate.toString() + "-08:00";
```

Comment:
I'm not sure what what the format of `expireDate.toString()` looks like on the server so I'm hardcoding "-08:00".  I'll replace "-08:00" with the appropriate offset that takes into account of daylight savings time in a subsequent commit.